src/Common/src/Interop/Windows/User32/Interop.GetSysColor.cs

```diff
@@ -8,7 +8,19 @@ internal partial class Interop
 {
     internal partial class User32
     {
-        [DllImport(Libraries.User32, SetLastError = true, CharSet = CharSet.Auto)]
-        public static extern int GetSysColor(int nIndex);
+        // The returned value is a COLORREF. The docs don't say that explicitly, but
+        // they do document the same macros (GetRValue, etc.). [0x00BBGGRR]
+        //
+        // This API sets last error, but we never check it and as such we won't take
+        // the overhead in the P/Invoke. It will only fail if we try and grab a color
+        // index that doesn't exist.
+        //
+        // If we *did* want to check the last error we would also have to *set* the
+        // last error as there is no way to determine if a return of 0 is the actual
+        // value (i.e. black) or the default. The API does not clear existing error
+        // codes if there is no error.
```

Comment:
> The API does not clear existing error codes if there is no error.

Don't we do that automatically now as part of P/Invokes with SetLastError?
https://github.com/dotnet/coreclr/pull/614

Reply:
Wow, missed that. :) Nice feature as there are a number of APIs that have this sort of gotcha (where they only set the last error some of the time).